### PR TITLE
[IA-1251] [IA-1461] Fix various test issues

### PR DESCRIPTION
**Pull request by rtitle**
See inline

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala [hunk 1]
```diff
@@ -69,7 +69,6 @@ final class LeonardoSuite extends Suites(
   new NotebookDataSyncingSpec,
   new NotebookGATKSpec,
   new NotebookHailSpec,
-  new NotebookLocalizeFileSpec,
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala]: Simply removing this test as it's no longer relevant. It failed in https://broadworkbench.atlassian.net/browse/IA-1380

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala [hunk 2]
```diff
@@ -59,6 +59,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
   lazy val ronEmail = ronCreds.email
 
   val clusterPatience = PatienceConfig(timeout = scaled(Span(10, Minutes)), interval = scaled(Span(20, Seconds)))
+  val clusterStopAfterCreatePatience = PatienceConfig(timeout = scaled(Span(20, Minutes)), interval = scaled(Span(20, Seconds)))
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala:62]: Adding longer patience for stop-after-creation. See https://broadworkbench.atlassian.net/browse/IA-1251

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala [hunk 3]
```diff
@@ -110,15 +110,19 @@ final class NotebookCustomizationSpec extends GPAllocFixtureSpec with ParallelTe
     "should populate user-specified environment variables" in  { billingProject =>
       implicit val ronToken: AuthToken = ronAuthToken
 
-      withNewCluster(billingProject, request = defaultClusterRequest.copy(customClusterEnvironmentVariables = Map("KEY" -> "value"))) { cluster =>
+      // Note: the R image includes R and python 3 kernels
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala:113]: Should fix https://broadworkbench.atlassian.net/browse/IA-1461.

Previously it was using the base image which doesn't have an R kernel (not sure how it ever passed).

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala (hunk 1 above)
Comment (rtitle) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala]: I added this back - the old localize is still used for legacy clusters.

**Review by Qi77Qi — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala (hunk 2 above)
Comment (kyuksel) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala:62]: Stopping a cluster taking longer than 10 mins sounds awfully long but I guess  🤷‍♂

**Review by kyuksel — APPROVED** [review]
👍

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala (hunk 2 above)
Comment (rtitle) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala:62]: In this instance creating it took 5mins and stopping it took 5 mins.

Comment (kyuksel) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala:62]: Oh the timeout is for both creating and stopping. That feels better :)

**Pull request merged**